src/commands/data/connectors/create.ts

```diff
@@ -5,7 +5,7 @@ import {cli} from 'cli-ux'
 import BaseCommand, {PostgresConnector} from '../../../lib/base'
 
 export default class ConnectorsCreate extends BaseCommand {
-  static description = 'create a new Data Connector'
+  static description = 'create a new Data Connector\nRead more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors'
```

Comment:
what happens when `h data:connectors --help` is run does it only show the first line?

Reply:
In oclif, the first line of the description goes at the top (eg: summary) and subsequent lines go in the `DESCRIPTION` heading.

```
ᐅ bin/run data:connectors --help                           
List all Data Connectors for a particular app or addon

USAGE
  $ heroku data:connectors

OPTIONS
  -a, --app=app        app to run command against
  -r, --remote=remote  git remote of app to use
  --addon=addon        The ID or name for the addon your your connector is attached to
  --json               Return the results as json
  --table              Return the results as a table

DESCRIPTION
  Read more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors

ALIASES
  $ heroku data:connectors:list

EXAMPLES
  heroku data:connectors -a your-app
  heroku data:connectors --app=your-app --json
  heroku data:connectors --addon=your-postgres-addon --table

COMMANDS
  data:connectors:create   create a new Data Connector
  data:connectors:destroy  Destroy a Data Connector
  data:connectors:info     Get information about a Data Connector
  data:connectors:pause    Pause change event creation on a Data Connector
  data:connectors:resume   Resume change event creation on a Data Connector
  data:connectors:update   Update the settings for a Data Connector
  data:connectors:wait     Wait for your Data Connector to be provisioned
```